### PR TITLE
Adds Shotgun Axe. Also tweaks shotgun sprites.

### PR DESCRIPTION
**Pull request by TravellingMerchant**
Shellguard Munitions Manufacturing is happy to bring you the latest in combat technology. Fortunately for you, this time, you're being given something a little less complex. 

Its a shotgun.

Its an axe. 

Its a god damn shotgun axe. 
Boasting a reinforced heat shielded barrel for strength and high carbon steel axe blade, this is true force to be reckoned with.

shotgun axe

Starting today, you can purchase this and its ammo from your operative uplink!
Now you can *disarm* that pesky Nanotrasen crew before they disarm you!

Technical specifications:
Nukie and hijack bartender shotgun.
Its a combat shotgun, but with melee capability.
Does 23 damage on hit, 18 on throw. Has a chance to embed. Acts as a sharp object.
Has a brutal custom free use hitsound. 
It is tube fed, meaning it is slower then the bulldog, it also costs more, at 15 TC compared to 10. 
Adds slug, buckshot and dragonsbreath speedloaders to the uplink as well.

Other shotguns have been tweaked to have a separate barrel and ammo tube at the end rather than one weird blob.
Before:
image

After
image

Source for the hitsound: http://www.freesound.org/people/EFlexTheSoundDesigner/sounds/376972/

Licensed for free use and to be edited. 

Here is a video, be warned, the sound is doubled for some apparent reason. It must have been my recording software. It is not like this in game.

https://www.youtube.com/watch?v=bZBvg-RbIMc&feature=youtu.be

It is a reference to Mack, from Marvel: Agents of Shield, but is not copyright by them. 

# EDIT: NOW ALSO A BARTENDER HIJACK ITEM

### CODE REVIEW

**Comment by KasparoVy** [comment]
Changelog?

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -372,6 +372,14 @@ var/list/uplink_items = list()
 	item = /obj/item/weapon/gun/projectile/automatic/pistol
 	cost = 4
 
+/datum/uplink_item/dangerous/shotchop
+	name = "L6 Squad Automatic Weapon"
+	desc = "A fully-loaded Aussec Armoury belt-fed machine gun. This deadly weapon has a massive 50-round magazine of devastating 7.62x51mm ammunition."
```
Comment (KasparoVy) [inline code/datums/uplink_item.dm]: This and the line above: Leftovers?

Comment (TravellingMerchant) [inline code/datums/uplink_item.dm]: Huh that isn't right.

**Comment by SamHPurp** [comment]
We about about 80 variations on 'gun'. 

Now this is "gun + axe"... while very Warhammer40k of you, I don't think we need, nor do I like the idea of. Sorry.

**Comment by TullyBurnalot** [comment]
As ludicrously OTP as this is, we should focus on more imaginative venues, as opposed to even more bladed implements or gun-related stuff.

**Comment by KasparoVy** [comment]
How about a gun blade rather than a gun with a blade that costs 420 tc?

**Comment by TravellingMerchant** [comment]
I mean, it's here. It's coded. Not like it takes anything to actually add in. It's not taking away from other resources. I'll see if I can also make it a barman hijack item too. 

It's my first actually item that I've coded without anyone else's help. More imaginative stuff can come after @TullyBBurnalot :D

**Comment by Anticept** [comment]
Sooooo... a shaxe?

**Comment by TravellingMerchant** [comment]
I prefer shotchop myself but a shaxe works too.

**Comment by MrFroztee** [comment]
I don't get why it has to be the bartender's item, other than that he owns a double barrel. 

Also, you'd probably be better off calling it "The Hunter" or "The Huntress". Names like Shotchop and Shaxe sort of just sound like you're trying to name a new sandwich.

**Comment by TravellingMerchant** [comment]
Official name is SG AX3, but its nickname at the moment is shotchop. 

Chose bartender because he currently does not have a bartender only hijack weapon, so he could do with one.

**Comment by MrFroztee** [comment]
Make it renamable like the DB and you're getting somewhere with that.

**Comment by TullyBurnalot** [comment]
@TravellingMerchant 

> I mean, it's here. It's coded. Not like it takes anything to actually add in. It's not taking away from other resources.

This is in no way a good argument to merge anything. You don't get to merge something just because "it's here" and doesn't conflict with anything. That's not how QA works.

**Comment by TravellingMerchant** [comment]
That isn't what I meant. I realise I misread what you said before.  I thought you were talking about how it was made. I was tired.

**Comment by HereticChurch** [comment]
This is actually a really fun, if simple item. I see no issue with having too many variations on guns, especially in Nuke Ops. People love variation and choice when it comes to loadouts, and something like this will probably see use.

Though... I wouldn't say this is an item that can single handedly help with a Hijack objective. Making it a bartender unique traitor item would be cool but it wouldn't help someone complete a hijack effectively without a lot of ammo and something to block the wave of tasers and lasers.

**Comment by TravellingMerchant** [comment]
Hmm, yeah. You have a point. Perhaps I should lower the cost for the hijack variant? It would be nice to have it available outside of nukies. Bartender is the best I could think of, but I'm welcome to ideas!

**Comment by outbools** [comment]
So how about ghetto shotchop involving improvised shotgun, wires/tape and axe?

**Comment by scrubmcnoob** [comment]
Why not have the axe be the butt of the gun and to swing it you grab the gun by it's body length
Sure when you're swinging you'll be pointing the barrel at yourself but it's not like anyone here knows gun safety.
Also the Axe looks kind of tacked on. Bayonets are fine, but a whole axe head would weigh the gun down quite a bit.
Also the Shaxe sounds interesting.

**Comment by TravellingMerchant** [comment]
It is modelled after the shotgun axe in agents of shield and this is also how they are made in real life too.

**Comment by Tayyyyyyy** [comment]
Outside the scope of this PR but this made me think about how it'd be cool if a bayonet system could be added. You'd be able to craft knives and stuff onto guns, at the cost of not being able to put them into a backpack. Would require a lot of work though, both coding and spriting.

**Comment by Fox-McCloud** [comment]
Don't feel this would be necessary; it doesn't really add much in terms of diversity to the nuke ops availability of items. It's just a gun with a higher force rating when you melee someone with it.

It's a bit of a "noob trap" item so to speak; it looks cool, it presents itself as being a viable synergy to melee and ranged, but the simple fact is, ranged exceeds melee objects even in close quarters/point blank, anyway, so it's actually encouraging bad gameplay on the player's part.

It'd have to be exceedingly strong for it to be decent at both, upon which it becomes the *optimal* gun to always select.

**Pull request closed without merging**